src/main/java/com/powsybl/network/map/model/LineMapData.java

```diff
@@ -25,6 +25,10 @@
 
     private String name;
 
+    private Boolean isTerminal1Connected;
+
+    private Boolean isTerminal2Connected;
+
```

Comment:
terminal1Connected and terminal2Connected for the fields names (and the getters are prefixed by 'is')